samtranslator/intrinsics/actions.py

```diff
@@ -522,7 +516,9 @@ def _check_input_value(self, value):  # type: ignore[no-untyped-def]
 
         return True
 
-    def _get_resolved_dictionary(self, input_dict, key, resolved_value, remaining):  # type: ignore[no-untyped-def]
+    def _get_resolved_dictionary(
+        self, input_dict: Optional[Dict[Any, Any]], key: str, resolved_value: Optional[Any], remaining: List[str]
```

Comment:
Are sure that `input_dict` is not `Dict[str, Any]`? Also can `resolved_value` be `Optional[str]`?

Reply:
Good catch. It should be Dict[str, Any]